namespaces/live-1.cloud-platform.service.justice.gov.uk/davidread-dev/00-namespace.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: davidread-dev
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HQ"
+    cloud-platform.justice.gov.uk/slack-channel: "davidread"
+    cloud-platform.justice.gov.uk/application: "Playing about with it"
+    cloud-platform.justice.gov.uk/owner: "Cloud Platform: david.read@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/cloud-platform-multi-container-demo-app"
+    cloud-platform.justice.gov.uk/team-name: "cloud-platform-test" 
```

Comment:
Team name in this instance would `Cloud Platform` and the owner would be `Cloud Platform: platforms@digital.justice.gov.uk`

Reply:
I've change the `owner` - I assume you weren't talking about `team-name` annotation, since that's GitHub team (without spaces).